kong/kong.lua

```diff
@@ -217,7 +217,12 @@ function Kong.balancer()
     -- record failure data
     addr.failures = addr.failures or {}
     local state, code = get_last_failure()
```

Comment:
move `local state, code = get_last_failure()` above `addr.failures = addr.failures or {}` it makes code more pleasing to read, :-).